### PR TITLE
[Merged by Bors] - implement `Reflect` for `Fxaa`

### PR DESCRIPTION
**Pull request by jakobhellermann**
# Objective

- public components and resources should implement `Reflect` if possible

## Solution

- derive `Reflect`

### CODE REVIEW

File: crates/bevy_core_pipeline/src/fxaa/mod.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@ mod node;
 
 pub use node::FxaaNode;
 
-#[derive(Eq, PartialEq, Hash, Clone, Copy)]
+#[derive(Reflect, Eq, PartialEq, Hash, Clone, Copy)]
```
Comment (MrGVSV) [inline crates/bevy_core_pipeline/src/fxaa/mod.rs:22]: Should both of these also derive `FromReflect`?

Comment (MrGVSV) [inline crates/bevy_core_pipeline/src/fxaa/mod.rs:22]: Nit: this should probably also add `#[reflect(PartialEq, Hash)]`

File: crates/bevy_core_pipeline/src/fxaa/mod.rs [hunk 2]
```diff
@@ -40,7 +40,8 @@ impl Sensitivity {
     }
 }
 
-#[derive(Component, Clone, ExtractComponent)]
+#[derive(Reflect, Component, Clone, ExtractComponent)]
+#[reflect(Component)]
```
Comment (MrGVSV) [inline crates/bevy_core_pipeline/src/fxaa/mod.rs:44]: Nit: we could also add `Default` to this list

**Comment by alice-i-cecile** [comment]
Seconding the above requests.

**Review by alice-i-cecile — APPROVED** [review]

**Review by MrGVSV — APPROVED** [review]

**Pull request closed without merging**